packages/wells/src/client/model/GeoJson.ts

```diff
@@ -0,0 +1,56 @@
+/* Types */
+
+export type Point2D = [number, number];
+
+export type GeoJson = Point | Polygon;
+
+/* Interfaces */
+
+export interface Geometry {
+  /**
+   * @type {string}
+   * @memberof GeoJson
+   */
+  id?: string;
+  /**
+   * @type {string}
+   * @memberof GeoJson
+   */
+  type?: string;
+  /**
+   * @type {any}
+   * @memberof GeoJson
+   */
+  properties?: any;
+  /**
+   * @type {Geometry}
+   * @memberof GeoJson
+   */
+  geometry: GeoJson;
+}
+
+export interface Point {
+  /**
+   * @type {string}
+   * @memberof Point
+   */
+  type: 'Point';
+  /**
+   * @type {Point2D}
+   * @memberof Point
+   */
+  coordinates: Point2D;
+}
+
+export interface Polygon {
+  /**
+   * @type {string}
+   * @memberof Polygon
+   */
+  type: 'Polygon';
+  /**
+   * @type {GeometryPoint[]}
+   * @memberof Geometry
+   */
+  coordinates: Point2D[][];
+}
```

Comment:
@kiril-me I guess you could add this into the geospatial-sdk and I can remove it from the wells sdk

Reply:
https://github.com/cognitedata/cognite-geospatial-sdk/pull/71/files

Reply:
Yea, I will update this sdk accordingly in a separate PR